modules/ipi-install-network-requirements.adoc

```diff
@@ -26,25 +28,166 @@ When using a VLAN, each NIC must be on a separate VLAN corresponding to the appr
+The API server must be able to resolve the worker nodes by the hostnames
+that are recorded in Kubernetes. If the API server cannot resolve the node
+names, then proxied API calls can fail, and you cannot retrieve logs from pods.
+====
+
+|Routes
+|`*.apps.<cluster_name>.<base_domain>.`
+|The wildcard A/AAAA record refers to the application ingress load balancer. The application ingress load balancer targets the nodes that run the Ingress Controller pods. The Ingress Controller pods run on the worker nodes by default. These records must be resolvable by both clients external to the cluster and from all the nodes within the cluster.
+
+For example, `console-openshift-console.apps.<cluster_name>.<base_domain>` is used as a wildcard route to the {product-title} console.
+
+|Provisioner node
+|`provisioner.<cluster_name>.<base_domain>.`
+|An A/AAAA record, and a PTR record, identify the provisioner node. These records must be resolvable by the nodes within the cluster.
+
+|Control plane nodes
+|`openshift-master-<n>.<cluster_name>.<base_domain>.`
+|An A/AAAA record and a PTR record identify each node for the control plane nodes. These records must be resolvable by the nodes within the cluster. The installer will create these records for you.
+
+|Compute nodes
+|`openshift-worker-<n>.<cluster_name>.<base_domain>.`
+|An A/AAAA record and a PTR record identify each node for the worker nodes. These records must be resolvable by the nodes within the cluster. The installer will create these records for you.
+
+|===
+
+[TIP]
+====
+You can use the `dig` command to verify DNS resolution.
+====
+
+[id="installation-dns-user-infra-example_{context}"]
+[discrete]
+== Example DNS configuration for installer-provisioned clusters
+
+This section provides A and PTR record configuration samples that meet the DNS requirements for deploying {product-title} on installer-provisioned infrastructure. The samples are not meant to provide advice for choosing one DNS solution over another.
+
+In the examples, the cluster name is `ocp4` and the base domain is `example.com`.
+
+
+.Example DNS A record configuration for an installer-provisioned cluster
+
+The following example is a BIND zone file that shows sample A records for name resolution in an installer-provisioned cluster.
+
+.Sample DNS zone database
+[%collapsible]
+====
+[source,text]
+----
+$TTL 1W
+@	IN	SOA	ns1.example.com.	root (
+			2019070700	; serial
+			3H		; refresh (3 hours)
+			30M		; retry (30 minutes)
+			2W		; expiry (2 weeks)
+			1W )		; minimum (1 week)
+	IN	NS	ns1.example.com.
+	IN	MX 10	smtp.example.com.
+;
+;
+ns1.example.com.		IN	A	192.168.1.5
+smtp.example.com.		IN	A	192.168.1.5
+;
+helper.example.com.		IN	A	192.168.1.5
+helper.ocp4.example.com.  IN	A	192.168.1.5
+;
+api.ocp4.example.com. IN	A	192.168.1.5 <1>
+*.apps.ocp4.example.com.	IN	A	192.168.1.5 <2>
```

Comment:
These should not be the same IP.

The same applies to the other records in this file too, but especially these two since they're referring to something specific within the cluster.